options/locale/locale_en-US.ini

```diff
@@ -1089,6 +1089,7 @@ projects.board.set_default = "Set Default"
 projects.board.set_default_desc = "Set this board as default for uncategorized issues and pulls"
 projects.board.delete = "Delete Board"
 projects.board.deletion_desc = "Deleting a project board moves all related issues to 'Uncategorized'. Continue?"
+projects.board.edit_color = "Color"
```

Comment:
I'm for a **generic** color at the beginning of the ini - we already have two similar values and dont need to translate the same thing over and over ...

Reply:
Did it, I removed all other color key, hope it was the right way